.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,75 @@
+name: 🐞 Bug Report
+description: Found a problem while using WiX?
+labels: [ "triage" ]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## FireGiant Support Customer?
+
+        If you are a [FireGiant Support Customer](https://www.firegiant.com/services/support-programs/) you can [submit bugs privately](https://support.firegiant.com/requests/new)
+
+        ---
+
+  - type: input
+    id: wixversion
+    attributes:
+      label: WiX Version
+      placeholder: '4.0.#'
+    validations:
+      required: true
+
+  - type: input
+    id: netversion
+    attributes:
+      label: .NET Version
+      placeholder: '6.0.0'
```

Comment:
This is probably too simple these days...What about asking for `dotnet --info` output instead?

Reply:
Not a bad idea, I was thinking about .NET Framework as well.

Reply:
I added MSBuild and Visual Studio since the goal is to get the "build system" version. I only added Visual Studio since I anticipate a lot of people never try to build from the command-line.